modules/core/src/lib/layer.js

```diff
@@ -679,23 +699,21 @@ export default class Layer extends Component {
 
   // Calculates uniforms
   drawLayer({moduleParameters = null, uniforms = {}, parameters = {}}) {
+    const currentProps = this.props;
     if (!uniforms.picking_uActive) {
-      this.updateTransition();
+      // Overwrite this.props during redraw to use in-transition prop values
+      this.props = this.updateTransition();
```

Comment:
Unrelated: The fact that `layer.props` can be changed is interesting. Wonder if applications could use that to subvert having to create new layers...